import/index.html

```diff
@@ -0,0 +1,298 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+  <link rel="icon" type="image/svg+xml" href="./favicon.svg" />
+  <meta charset="utf-8">
+  <title>Turnkey Import</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {
+    textarea {
+      width: 400px;
+      margin: 0.5em;
+      font-size: .875rem;
+      line-height: 1.25rem;
+      height: 120px;
+      color: #555b64;
+      border: none;
+      resize: none;
+      word-wrap: break-word;
+      overflow-wrap: break-word;
+      border: none;
+      font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    }
+    textarea:focus {
+      outline: none;
+      border: none;
+    }
+  }
+  </style>
+</head>
+
+<body>
+  <form>
+    <textarea name="wallet-bundle" id="wallet-bundle"></textarea>
+  </form>
+
+  <!--
+    Script defining important helpers.
+    These helpers are unit-testable, so most of the logic should be written here.
+  -->
+  <script>
+    window.TKHQ = function() {
+      /** constants for LocalStorage */
+      const TURNKEY_TARGET_EMBEDDED_KEY = "TURNKEY_TARGET_EMBEDDED_KEY"
+      /** 1 week in milliseconds */
+      const TURNKEY_TARGET_EMBEDDED_KEY_TTL_IN_MILLIS = 1000 * 60 * 60 * 24 * 7;
```

Comment:
oops i actually made it really long at first because we don't really need it to expire, but meant to eventually remove the expiration entirely. the reason is that this `target_embedded_key` is the public key of the encryption key we're storing on the org. and we're not setting any expiration there.